lib/api.v2.js

```diff
@@ -123,26 +123,27 @@ function _selectFields(fields, models) {
 // structure the response according to v2 schema and pass through to _selectFields
 function _formatResponse(query, models) {
 
-  var _formatAssets = function formatAssets(model) {
-    return _.map(model.assets, function (asset) {
+  function _formatModel(model, excludeAssets) {
+    model.assets = excludeAssets ? void 0 : _.map(model.assets, asset => {
       var obj = {};
       obj[asset.version] = asset;
       delete asset.version;
```

Comment:
Do we want to delete this? It's redundant given the structure of assets, but it does offer some clarity.